playwright/pages/basePage.ts

```diff
@@ -110,11 +112,11 @@ export class BasePage {
   }
 
   async expectSuccessBanner() {
-    await this.page.waitForLoadState("networkidle");
-    await Promise.all([
-      this.successBanner.first().waitFor({ state: "visible", timeout: 15000 }),
-      expect(this.errorBanner, "No error banner should be visible").not.toBeVisible(),
-    ]);
+    await this.successBanner.first().waitFor({
+      state: "visible",
+      timeout: SUCCESS_BANNER_TIMEOUT,
+    });
+    await expect(this.errorBanner, "No error banner should be visible").not.toBeVisible();
```

Comment:
Is it possible to have success and the error right after? Isn't enough to confirm that just success pops up?

Reply:
> Is it possible to have success and the error right after?
It's hard to say, I think it's possible for two different operations to happen at the same time. 

> Isn't it enough to confirm that just success pops up?
It might not be enough.